### PR TITLE
 feat: UI display contract function broadcast data

### PR DESCRIPTION
**Pull request by FilipHarald**

### CODE REVIEW

**Comment by FilipHarald** [comment]
@Mautjee I know we should have just one `OptionButtons` I made separate issue for it: https://github.com/orgs/aztlan-labs/projects/1/views/1?pane=issue&itemId=85692639

File: services/explorer-ui/src/pages/contract-class-details/constants.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  | "contractInstances"
+  | "privateFunctions"
+  | "unconstrainedFunctions";
+
+export const tabIds = [
+  "contractVersions",
+  "contractInstances",
+  "privateFunctions",
+  "unconstrainedFunctions",
+];
```
Comment (FilipHarald) [inline services/explorer-ui/src/pages/contract-class-details/constants.ts:14]:
```suggestion
]  as const;
```

File: services/explorer-ui/src/pages/contract-class-details/tabs.tsx
[the remark this answers is not in the document]
Comment (FilipHarald) [reply services/explorer-ui/src/pages/contract-class-details/tabs.tsx:1]: Haha, see this comment :point_up: https://github.com/aztlan-labs/chicmoz/pull/180#issuecomment-2452452355

[the remark this answers is not in the document]
Comment (Mautjee) [reply services/explorer-ui/src/pages/contract-class-details/tabs.tsx:1]: aaahh nice oke! then its fine. I added some logic to it for so it shows the selected one so will need to add it here to

**Pull request merged**